services/src/main/java/org/fao/geonet/api/reports/ReportInternalMetadata.java

```diff
@@ -67,7 +89,7 @@ public void create(final ServiceContext context,
             csvFilePrinter.println();
 
             String[] entries = (
-                "Metadata ID#Metadata Title#Metadata Create date#"
+                "Metadata ID#Metadata Title#Metadata Create date#Metadata Create date#"
```

Comment:
The new column should be titled `Metadata Update date`

Reply:
Fixed in https://github.com/geonetwork/core-geonetwork/pull/6082/commits/33dea629a4816cd98a98a7f13cde5a6f8038aa89